torch_geometric/explain/explainer.py

```diff
@@ -203,3 +203,54 @@ def _threshold(self, explanation: Explanation) -> Explanation:
             explanation[key] = mask
 
         return explanation
+
+    def visualize_feature_importance(
+        self,
+        explanation: Explanation,
+        index: int,
+        feat_labels: Optional[List[str]] = None,
+        n_features: int = 20,
+    ):
+        r"""Creates a bar plot of the node features importance given
+        a :attr:`node_feat_mask`
+
+        Args:
+            node_feat_mask (Tensor): The node feature mask to plot.
+            feat_labels (List[str], optional): Optional labels for features.
+            n_features (int, optional): Top N features to plot.
+                (default: :int:`20`)
+        :rtype: :class:`matplotlib.axes.Axes`
+        """
+        import matplotlib.pyplot as plt
+        import pandas as pd
+
+        node_feature_explanation = explanation.node_feat_mask[index]
+        node_feature_explanation = node_feature_explanation.numpy()
```

Comment:
```suggestion
        node_feature_explanation = node_feature_explanation.cpu().numpy()
```
In case `node_feature_explanation` is on GPU.